python/cudf/cudf/utils/dtypes.py

```diff
@@ -16,6 +16,7 @@
 from cudf.core.missing import NA
 
 _NA_REP = "<NA>"
+_NAT_REP = "NaT"
 
```

Comment:
Do we need to do something about this line: https://github.com/galipremsagar/cudf/blob/09466815b0d4da61b80115de4e0612965ccde82a/python/cudf/cudf/utils/dtypes.py#L643

Reply:
Yup, updated it.